### PR TITLE
test/extended: Properly wait for project/namespace to be provisioned and fix deployer role

### PR DESCRIPTION
**Pull request by tnozicka**

### CODE REVIEW

**Comment by bparees** [comment]
/lgtm

aws-e2e-builds is currently broken, so at a minimum this will need to wait until that issue is resolved. (we're working on it).

**Comment by bparees** [comment]
/lgtm cancel
per @soltysh's request to see this pass a few times.

**Comment by tnozicka** [comment]
/hold
(for the few passes, when the CI gets back up)

File: pkg/cmd/server/bootstrappolicy/policy.go [hunk 1]
```diff
@@ -539,7 +539,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get").Groups(kapiGroup).Resources("pods/log").RuleOrDie(),
 				rbacv1helpers.NewRule("create", "list").Groups(kapiGroup).Resources("events").RuleOrDie(),
 
-				rbacv1helpers.NewRule("update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
+				rbacv1helpers.NewRule("create", "update").Groups(imageGroup, legacyImageGroup).Resources("imagestreamtags").RuleOrDie(),
```
Comment (tnozicka) [inline pkg/cmd/server/bootstrappolicy/policy.go]: @openshift/sig-auth what did we get with 1.12 rebase that could have caused the need for this change? @bparees or did something changed wrt to imagestream?

@soltysh (this is the actual fix for the rebase revert)

**Comment by tnozicka** [comment]
As in the end these turned out to be 2 independent fixes I've split the deployer permissions update into https://github.com/openshift/origin/pull/21958

File: pkg/cmd/server/bootstrappolicy/policy.go (hunk 1 above)
Comment (bparees) [reply pkg/cmd/server/bootstrappolicy/policy.go]: nothing has changed related to imagestream permissions from our side.

**Comment by tnozicka** [comment]
ci seems to be back, going for few runs
/test e2e-aws

**Comment by tnozicka** [comment]
(testing second pass)
/test e2e-aws

**Comment by tnozicka** [comment]
(testing third pass)
/test e2e-aws

**Comment by tnozicka** [comment]
(storage issues - https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/21905/pull-ci-openshift-origin-master-e2e-aws/3867)

(testing fourth pass)
/test e2e-aws

**Comment by bparees** [comment]
/hold

aws-builds is having issues that need to be sorted out.

**Pull request merged**